src/colorizer/RecordingControls.ts

```diff
@@ -1,110 +1,121 @@
-import ColorizeCanvas from "./ColorizeCanvas";
+export type RecordingOptions = {
+  /** Frame to start recording from, inclusive. 0 by default. */
+  min: number;
+  /** Highest frame number to record, inclusive. 0 by default. */
+  max: number;
```

Comment:
Should start take in the range as required parameters rather than making them part of the options? This feels a bit cleaner to me but thought I should ask.

Reply:
My take: I don't think it's too different either way.  We can always change it later if there's a good reason.